### PR TITLE
London Class 7 - Oksana Kostenko - JavaScript Core 3 - Week 2

### PR DESCRIPTION
**Pull request by oksygenn**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:  Oksana Kostenko
- Your City: London
- Your Slack Name:  Oksana Kostenko

# Homework Details

- Module: JS Core 3 
- Week: 2

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

### CODE REVIEW

File: challenges/dog-photo-gallery/script.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
+const URL = "https://dog.ceo/api/breeds/image/random";
+
+const setup = () => {
```
Comment (Laura-TM) [inline challenges/dog-photo-gallery/script.js:3]: Your JS file is really organised, I want to try that on mine! Perhaps one thing you could do is add some CSS to give it a nice outlook (e.g. remove the bullet points)

**Comment by Laura-TM** [comment]
Your work is very organised, well done! Just made a small suggestion within your code :-)

File: challenges/dog-photo-gallery/script.js (hunk 1 above)
Comment (oksygenn) [reply challenges/dog-photo-gallery/script.js:3]: Thanks Laura. I'll add CSS later 👍

File: challenges/programmer-humour/script.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    })
+    .catch((error) => {
+      const h1 = document.createElement("h1");
+      h1.innerText = `Oops! Something went wrong :( ${error}`;
+      h1.style.color = "red";
+      body.append(h1);
+    });
+};
+
+window.onload = getPicture;
```
Comment (ybayraktutan) [inline challenges/programmer-humour/script.js:26]: Hi Oksana, your code is very well organised.  Keep up the good work!

Comment (oksygenn) [reply challenges/programmer-humour/script.js:26]: Thank you Yasemin :)

**Pull request closed without merging**